client/src/test/kotlin/io/spine/examples/pingh/client/e2e/PersonalInteractionTest.kt

```diff
@@ -115,21 +123,28 @@ public class PersonalInteractionTest : IntegrationTest() {
      * 6. Reads mentions that were updated in the first session.
      */
     @Test
-    public fun `the user should log in, log out, log in again, and then gets mentions`() {
+    internal fun `the user should log in, log out, log in again, and then gets mentions`() {
         client().logOut()
+        delay()
         shouldThrow<IllegalStateException> {
             client().findUserMentions()
         }
         client().logIn(username)
+        delay()
```

Comment:
I don't understand why we cannot subscribe to events instead of this and the previous `delay()`s.